network-api/networkapi/templates/buyersguide/fragments/consumer_creepometer_page_2023/outro.html

```diff
@@ -0,0 +1,28 @@
+<div class="{{ wrapper_class }} tw-bg-gradient-to-b tw-from-white tw-to-purple-05">
+  <div class="tw-container">
+    <div class="tw-grid tw-grid-cols-1 large:tw-grid-cols-3 tw-gap-x-6 tw-gap-6 large:tw-gap-y-0 tw-mb-[120px]">
+      <div class="tw-col-span-1 tw-rounded-2xl tw-bg-white">
+        <div class="tw-h-[200px] large:tw-h-full tw-w-full tw-rounded-2xl tw-text-center tw-bg-no-repeat tw-bg-[center_-8px] large:tw-bg-center tw-bg-[length:auto_300px] medium:tw-bg-cover tw-bg-[url('../_images/buyers-guide/consumer-creepometer/our-approach.png')]"></div>
+      </div>
+      <div class="tw-col-span-2 tw-p-14 medium:tw-p-24 tw-bg-white tw-rounded-2xl">
+        <h2 class="{{ h2_class }}">Our Approach</h2>
+        {% with p_class="tw-text-gray-80" %}
+        <p class="{{ p_class }}">The Annual Consumer Creep-O-Meter is calculated using both quantitative and qualitative measures. Our researchers average together findings from our 2023 and late 2022 *Privacy Not Included editions, each of which entails hundreds of hours of research. In 2023 and late 2022, we published three editions and one research report covering 172 different products, from cars and mental health apps to fitness trackers and kids toys.</p>
```

Comment:
Yea.. I would have gone with SCSS if the compilation time isn't like 20x times longer than Tailwind. Since this page won't be updated frequently after launch (or at all), I went with Tailwind.

Reply:
That's totally reasonable, thanks for clarifying!